modules/caddyhttp/fileserver/staticfiles.go

```diff
@@ -172,6 +172,9 @@ type FileServer struct {
 	fsmap caddy.FileSystems
 
 	logger *zap.Logger
+
+	// FileLimit limits the number of up to n DirEntry values in directory order.
+	FileLimit int `json:"file_limit,omitempty"`
```

Comment:
This should go inside the `Browse` struct, since it's specifically an option for the browse feature. Look at the `reveal_symlinks` option and how that's wired up, it should be a sibling of that.

Reply:
Updated.